src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -788,6 +788,76 @@
 short-summary: Show a deployment operation at tenant scope.
 """
 
+helps['deployment-scripts'] = """
+type: group
+short-summary: Manage deployment scripts at subscription or resource group scope.
+"""
+
+helps['deployment-scripts list'] = """
+type: command
+short-summary: Lists all deployment scripts.
```

Comment:
Please use active voice.
```suggestion
short-summary: List all deployment scripts.
```